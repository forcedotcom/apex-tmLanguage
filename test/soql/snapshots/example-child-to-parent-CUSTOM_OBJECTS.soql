SELECT Id, FirstName__c, Mother_of_Child__r.FirstName__c FROM Daughter__c WHERE Mother_of_Child__r.LastName__c LIKE 'C%'
