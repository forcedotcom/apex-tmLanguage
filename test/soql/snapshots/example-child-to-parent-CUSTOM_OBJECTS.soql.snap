>SELECT Id, FirstName__c, Mother_of_Child__r.FirstName__c FROM Daughter__c WHERE Mother_of_Child__r.LastName__c LIKE 'C%'
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^ source.soql keyword.query.field.apex
#         ^ source.soql punctuation.separator.comma.apex
#          ^ source.soql
#           ^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                       ^ source.soql punctuation.separator.comma.apex
#                        ^ source.soql
#                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                                        ^ source.soql
#                                                         ^^^^ source.soql keyword.operator.query.from.apex
#                                                             ^ source.soql
#                                                              ^^^^^^^^^^^ source.soql storage.type.apex
#                                                                         ^ source.soql
#                                                                          ^^^^^ source.soql keyword.operator.query.where.apex
#                                                                               ^ source.soql
#                                                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                                                                                              ^ source.soql
#                                                                                                               ^^^^ source.soql keyword.operator.query.apex
#                                                                                                                   ^ source.soql
#                                                                                                                    ^ source.soql string.quoted.single.apex punctuation.definition.string.begin.apex
#                                                                                                                     ^^ source.soql string.quoted.single.apex
#                                                                                                                       ^ source.soql string.quoted.single.apex punctuation.definition.string.end.apex
>